.travis.yml

```diff
@@ -82,6 +82,7 @@ matrix:
       env:
         - SPLIT="4/4"
 before_install:
+  - pip uninstall -y setuptools
```

Comment:
See here: https://github.com/darshanime/sympy/blob/disutils-import/.travis.yml#L120

Installation is only tested if the `$TEST_SAGE` environment variable is `true`. In this install block, you could do this:

1. remove setuptools
2. install without setuptools
3. uninstall sympy
4. install setuptools
5. install with setuptools
6. run sage tests